### DISCUSSION TITLE
Special budget

### DISCUSSION THREAD

**Discussion by amitamitamitamit** [firefly-iii/firefly-iii#7038] [Firefly III] [answered]
Hey,
Can I create a rollover budget with maximum positive number which can go negative?
For example, if I set a monthly budget of $25 per month with a maximum of $40:
 * Jan - $25
 * Feb - $40 (because it is the maximum)
Spending $80 in Mar
 * Mar - ($15) ($80-$40+$25)
* Apr - $10

**Comment by JC5** [comment, marked as answer]
I'm sorry, there is no maximum for budgets. If you have money left, save it using piggy banks for example.
